renovate.json

```diff
@@ -28,6 +28,17 @@
       "depNameTemplate": "com.google.protobuf:protobuf-java",
       "datasourceTemplate": "maven"
     },
+    {
+      "customType": "regex",
+      "fileMatch": [
+		"^\\.cloudbuild/library_generation/library_generation\\.Dockerfile$"
+      ],
+      "matchStrings": [
+        "ARG PROTOC_VERSION==(?<currentValue>.+?)\\n"
+      ],
+      "depNameTemplate": "com.google.protobuf:protobuf-java",
```

Comment:
The version of `com.google.protobuf:protobuf-java` is not the same as protoc.

Reply:
Thanks for the catch. I corrected it to use the github releases